napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -336,9 +355,10 @@ def _mark_conflicts(self, new_shortcut, row) -> bool:
 
```

Comment:
I think is there to point out where the row of the conflicting action (in case it is available over the current keybindings table being shown). So for example:

![imagen](https://github.com/user-attachments/assets/5f8979f6-d9c1-4a81-9b92-dcc7147f1eab)

If you remove it then you get something like:

![imagen](https://github.com/user-attachments/assets/2115fc37-b428-4d2d-add8-4b352e173aea)

Probably giving it a name like `conflict_action_row` could make things a little bit clearer 🤔

Reply:
Agreed, let's rename it while we're here to `conflicting_row` or similar

Reply:
I think what makes this more confusing now is that we are not just returning all the actions of the selected layer, we are also returning the viewer actions, which means that `row1` can be a number that is greater than the number of rows in the current table.

It still works fine (I think) but the code is not ideal now...

Reply:
Yep that is totally right! In fact, this code is working because the firts group of keybindings that is returned from the utility method to get the actions is always the current selected one (so the index generated by the `enumerate` call makes actions match the table row count) and, as mentioned, seems like passing a row value greater than the number of rows displayed is just ignored. Maybe being explicit about passing `[row, row1]`/`[row, conflicting_row]` or `[row]` could make things clearer? So something like:

```python
rows_with_warning = [row]
# put warning on conflicting row since is part of the current table being displayed
if self._table.rowCount() <= conflicting_row:
    rows_with_warning.append(conflicting_row)

# show warning symbols
self._show_warning_icons(rows_with_warning)
```

Reply:
Hmm I don't know if it's worth adding the extra code when current code works (or is there a possibility of a greater row value not being ignored and causing a problem?).
I'm okay either way. Maybe @DragaDoncil! can weigh in?

Reply:
For now I just change the variable name: `row1` -> `conflicting_row`. But let me know if further changes make sense!